bin/write-sources.js

```diff
@@ -22,8 +20,8 @@ function dealWithCrossWalk(crosswalk) {
 const handleSource = async (sourceName) => {
   await makeDir(exportPath);
 
-  const country = (await import(path.join(__root, `/sources/${sourceName}`)))
-    .default;
+  const importURL = new URL(`/sources/${sourceName}`, ROOT_URL);
+  const country = (await import(importURL)).default;
```

Comment:
I'm getting a error on this line because `importURL` is not a file that can't be found.

`2022-04-28 22:03:34 error: CATCH Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/sources/argentina.js' imported from /Users/jzhen/Dev/waterthetrees/wtt_area/bin/write-sources.js`